public/app/plugins/datasource/phlare/dataquery.gen.ts

```diff
@@ -0,0 +1,38 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     TSTypesJenny
+//     PluginTSTypesJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+import * as common from '@grafana/schema';
+
+export const DataQueryModelVersion = Object.freeze([0, 0]);
+
+/**
+ * Specify the query flavor
```

Comment:
Is this comment from the `common.DataQuery`?

Reply:
Yes, it's gone now